Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -2591,6 +2610,62 @@ def suppress_alert_command(client: Client, args: Dict[str, Any]) -> CommandResul
     )
 
 
+def get_alert_group_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+
+    returns the alert group id
+
+    """
+    agent_id = str(args.get('agentId', ''))
+    alert_id = args.get('alertId')
+
+    try:
+        alert_group_ids = client.get_alert_group(agent_id)
+    except Exception as e:
+        if '404' in str(e):
+            raise ValueError(f"Alerts of host {agent_id} Not Found")
+
+    alert_groups = {}
+    for entry in alert_group_ids['data']['entries']:
+        alert_groups.update({str(entry['last_alert']['_id']): entry['_id']})
+    return alert_groups[alert_id]
+
+
+def acknowledge_alert_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+
+    returns a success message to the war room
+
+    """
+    agent_id = str(args.get('agentId', ''))
+    alert_id = args.get('alertId')
+    comment = str(args.get('comment', ''))
+
+    try:
+        alert_group_ids = client.get_alert_group(agent_id)
+
+        alert_groups = {}
+        for entry in alert_group_ids['data']['entries']:
+            alert_groups.update({str(entry['last_alert']['_id']): entry['_id']})
+
+        alert_group_id = alert_groups[alert_id]
+
+        client.acknowledge_alert_request(alert_group_id, comment)
+    except KeyError:
+        raise KeyError(f'The Alert {alert_id} for Host {agent_id} - Not Found')
+    except Exception as e:
+        if '404' in str(e):
+            raise ValueError(f'The Alert {alert_id} with Alert Group {alert_group_id} for Host {agent_id} - Not Found')
+        else:
+            raise ValueError(e)
+
+    # no exceptions raised->successful request
+    return CommandResults(
+        readable_output=f'The Alert {alert_id} with Alert Group {alert_group_id} for Host {agent_id} got'
+        'acknowledged successfully.'
+    )
```

Comment:
This CommandResults is missing many parameters what about the outputs you did specify in the command yml that you have outputs. This should be passed to the outputs in the command results. also
 outputs_prefix and optionaly outputs_key_field please look at other functions using the CommandResults.